docs/docs/getting-started/docker-sidecar.md

```diff
@@ -0,0 +1,51 @@
+# Create a Docker Sidecar Function
+
+Lug Docker Sidecar functions are Lug Hybrid functions with an extra benefit: they attach a running Docker container to 
```

Comment:
"Lug Docker Sidecar" feels clunky. Maybe drop docker since that comes up as the extra benefit.

Reply:
Eh, I think this might be necessary complexity in the naming. "Lug Sidecar" could be a lot of things